wolfcrypt/src/sp_dsp32.c

```diff
@@ -4538,21 +4538,21 @@ void wc_ecc_fp_free(void)
 }
 
 
-AEEResult wolfSSL_open(const char *uri, remote_handle64 *handle) 
+AEEResult wolfSSL_open(const char *uri, remote_handle64 *handle)
 {
   /* can be any value or ignored, rpc layer doesn't care
    * also ok
    * *handle = 0;
    * *handle = 0xdeadc0de;
    */
-   *handle = (remote_handle64)malloc(1);
+   *handle = (remote_handle64)XMALLOC(1, NULL, DYNAMIC_TYPE_ECC);
```

Comment:
@JacobBarthelmeh are these DSP changes okay? 
@douzzer why `DYNAMIC_TYPE_ECC`?

Reply:
yeah, that is ok to change

Reply:
Should the handle really be only 1 byte? Should it be at least the size of a pointer? Can we choose a better dynamic type?

Reply:
It's ok to adjust it to the size of a pointer if wanted. Maybe on the dynamic types it could be  DYNAMIC_TYPE_FILE or DYNAMIC_TYPE_NONE or a new DYNAMIC_TYPE such as DYNAMIC_TYPE_HW_HANDLE